backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,49 @@
+## Port
+server.port = 5000
+endpoints.env.enabled=true
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
+## MySQL
+spring.datasource.username = ${SECRETS_DB_USERNAME}
+#spring.datasource.username = omegaadmin@omegadatabase3
```

Comment:
Is this real or a dummy? We should remove it if it's real

Reply:
This is an environment variable the cluster gives to the container. secret of DB.